src/OneOff.php

```diff
@@ -45,6 +45,7 @@ public function getEnd(): ?DateTimeInterface
 
     public static function fromJson(string $json): Frequency
     {
+        /** @var string $data */
```

Comment:
same here

Reply:
```php
  Line   OneOff.php                                                            
 ------ ---------------------------------------------------------------------- 
  50     Parameter #1 $datetime of class DateTime constructor expects string,  
         mixed given.  
```

Reply:
Same here, throw an error if it's not string.